### PR TITLE
Django upgrade

### LINKED ISSUE #23: Upgrade Django to 2.0.11
[CVE-2019-6975](https://nvd.nist.gov/vuln/detail/CVE-2019-6975)
moderate severity
Vulnerable versions: >= 2.0.0, < 2.0.11
Patched version: 2.0.11

Django 1.11.x before 1.11.19, 2.0.x before 2.0.11, and 2.1.x before 2.1.6 allows Uncontrolled Memory Consumption via a malicious attacker-supplied value to the django.utils.numberformat.format() function.

[CVE-2019-3498](https://nvd.nist.gov/vuln/detail/CVE-2019-3498)
More information
low severity
Vulnerable versions: >= 2.0.0, < 2.0.10
Patched version: 2.0.10

In Django 1.11.x before 1.11.18, 2.0.x before 2.0.10, and 2.1.x before 2.1.5, an Improper Neutralization of Special Elements in Output Used by a Downstream Component issue exists in django.views.defaults.page_not_found(), leading to content spoofing (in a 404 error page) if a user fails to recognize that a crafted URL has malicious content.

### PR DESCRIPTION
**Pull request by storymode7**
Upgrade Django version to 2.2.3 and add python 3.7.3 along with 3.6.5
This fixes #23

### CODE REVIEW

File: nexus/users/tests/test_models.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ class UserModelTestCase(TestCase):
     def test_create_user(self):
         u = User.objects.create_user(
             email='f@f.com', password='abc', first_name="F", last_name='B',
-            gender='others', tshirt_size='large', phone_number='+910123456789'
+            gender='others', tshirt_size='large', phone_number='+919876543210'
```
Comment (CuriousLearner) [inline nexus/users/tests/test_models.py]: Why this phone number change?

Comment (CuriousLearner) [reply nexus/users/tests/test_models.py]: Is it because of validations of PhoneNumberField?

Comment (storymode7) [reply nexus/users/tests/test_models.py]: Yes. The previous versions ignored if the number was not valid. This change was introduced in a 3.0.1. 
https://github.com/stefanfoulis/django-phonenumber-field/commit/b3ff5c387ace44203cf646c6513aadcd893fcc38

File: nexus/users/migrations/0004_auto_20190723_1943.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.3 on 2019-07-23 19:43
```
Comment (CuriousLearner) [inline nexus/users/migrations/0004_auto_20190723_1943.py]: Why is this new migration generated here?

Comment (storymode7) [reply nexus/users/migrations/0004_auto_20190723_1943.py]: This is not a new migration actually. It was missed previously when the `is_core_organizer` and `is_volunteer` were edited to allow `blank` values.

Comment (storymode7) [reply nexus/users/migrations/0004_auto_20190723_1943.py]: This migration is not related to Django upgrade. It is just a missed migration from a previous PR.

File: provisioner/roles/project_data/tasks/webpack.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
```
Comment (CuriousLearner) [inline provisioner/roles/project_data/tasks/webpack.yml]: Webpack is not needed in here, I guess

File: requirements/development.txt [hunk 4]
```diff
@@ -28,8 +28,8 @@ bumpversion==0.5.3
 # Deployment and Tasks
 # -------------------------------------
 fabric3==1.14.post1
-ansible==2.5.5
+ansible==2.8.3
```
Comment (CuriousLearner) [inline requirements/development.txt]: This is risky. Have you tested that it works with this change?

**Pull request merged**